extensions/applicationinsights-clickanalytics-js/src/ClickAnalyticsPlugin.ts

```diff
@@ -147,6 +148,12 @@ export class ClickAnalyticsPlugin extends BaseTelemetryPlugin {
                     _contentHandler = new DomContentHandler(_config, logger);
                     let metaTags = _contentHandler.getMetadata();
                     _pageAction = new PageAction(_self, _config, _contentHandler, _config.callback.pageActionPageTags, metaTags, logger);
+
+                    const defaultElementTypes = "A,BUTTON,AREA,INPUT";
```

Comment:
Why not just initialize this as the "default" config?

And them move the function below to here (forcing uppercase and trimming ) -- the `clickCaptureElements` variable